website/content/en/docs/contribution-guidelines/local-docs.md

```diff
@@ -24,7 +24,8 @@ $ git submodule update --init --recursive
 
 ## Build and Serve
 
-You can build and serve your docs to localhost:1313 with:
+You can build and serve your docs to localhost:1313. From the `website`
+directory run:
 
```

Comment:
The links in https://deploy-preview-2783--operator-sdk.netlify.com/docs/contribution-guidelines/developer-guide/ are not working.

Reply:
Should be fixed. There are still a lot of broken links all over which I plan to fix along with the new link checker.